### PR TITLE
fix: add example config and pipeline for sonar

### PR DESCRIPTION
**Pull request by minzcmu**
This pipeline only has an example config and the a pipeline to update sonarqube image.
The actual config is stored internally.

Related to: https://github.com/screwdriver-cd/screwdriver/issues/1007

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,82 @@
 # sonar-pipeline
-Pipeline to deploy sonarqube server to kubernetes
+Pipeline to deploy sonarqube server to kubernetes. Currently it only supports updating sonarqube's image. Other configuration requires manual update via helm.
+
+## First Time Deployment
+To simplify the deployment, we are using helm, the package manager for k8s, to bundle all we need for sonar and deploy them all in once. Example `values.yaml` can be found under `config-example` folder. The actual config files are stored somewhere else secretly.
```
Comment (tkyi) [inline README.md]: and deploy them all `at` once

File: README.md [hunk 2]
```diff
@@ -1,2 +1,82 @@
...
+
+    ==> v1/PersistentVolumeClaim
+    beta-postgresql   
+    beta-sonarqube
+    ```
+
+- Manual update in sonar UI:
+
+    For the following config, it's not configurable during boot time. Need to do it after sonar is up and running.
+  1. Go to **${your-sonar-host}** and log in as `admin` with default password `admin`.
```
Comment (tkyi) [inline README.md]: oh boy

**Review by tkyi — APPROVED** [review]
Minor comment, otherwise LGTM 👍

**Review by pranavrc — APPROVED** [review]
image

File: README.md (hunk 1 above)
Comment (minzcmu) [reply README.md]: addressed

File: README.md (hunk 2 above)
Comment (minzcmu) [reply README.md]: I know... They put it as a future improvement. https://github.com/SonarSource/docker-sonarqube/blob/master/recipes.md

Comment (tkyi) [reply README.md]: 😢

**Pull request merged**